src/activities/components/ActivityList.vue

```diff
@@ -246,3 +273,10 @@ export default {
   },
 }
 </script>
+
+<style scoped lang="stylus">
+.date-with-day-name
+  font-weight 700
+  color rgba(0, 0, 0, 0.7)
+  text-align center
```

Comment:
You can use quasar classes `text-center` and `text-bold` here (we like to use the quasar classes where possible, which you've done in most places :+1:)

(and as there would then be only 1 CSS value left, could perhaps be inline in a style attribute?)

Reply:
done